hook.php

```diff
@@ -80,6 +80,8 @@ function plugin_escalade_install() {
          `remove_delete_assign_supplier_btn`       TINYINT(1) NOT NULL DEFAULT 1,
          `use_filter_assign_group`                 INT(11) NOT NULL,
          `ticket_last_status`                      INT(11) NOT NULL,
+         `remove_requester`                        INT(11) NOT NULL,
+         `use_assign_requester_group`              INT(11) NOT NULL,
```

Comment:
This fields does not seems to be used.

Reply:
I am correcting this, I had planned to add a group deletion but I changed my mind because it already exists in the Beaviours plugin.